apps/src/templates/teacherDashboard/StatsTableWithData.jsx

```diff
@@ -0,0 +1,56 @@
+import React, {Component, PropTypes} from 'react';
+import Spinner from '@cdo/apps/code-studio/pd/components/spinner';
+import StatsTable from './StatsTable';
+
+class StatsTableWithData extends Component {
+  static propTypes = {
+    sectionId: PropTypes.string,
+  };
+
+  state = {
+    studentsCompletedLevelCount: {},
+    section: {},
+    isLoading: true
+  };
+
+  componentDidMount() {
+    const completedLevelsUrl = `/dashboardapi/sections/${this.props.sectionId}/students/completed_levels_count`;
+    $.ajax({
```

Comment:
also not a refactor for right now, but i think we could move both of these AJAX requests into redux